utilities/src/main/java/io/syndesis/qe/utils/GoogleSheetsUtils.java

```diff
@@ -0,0 +1,111 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+public class GoogleSheetsUtils {
+    @Autowired
+    private GoogleAccounts accounts;
+    private Sheets sheets;
+    public static String TEST_SHEET_ID = "";
+    public static String DATA_TEST_SPREADSHEET = "1_OLTcj_y8NwST9KHhg8etB10xr6t3TrzaFXwW2dhpXw";
+
+
+    @PostConstruct
+    private void initSheets() {
+        if (sheets == null) {
+            sheets = accounts.getGoogleAccountForTestAccount("QE Google Sheets").sheets();
+        }
+    }
+
+    public List<String> getSpreadSheetValues() {
+        return getSpreadSheetValues(TEST_SHEET_ID, "A1:E3");
+    }
+
+    public void clearSpreadSheetValues(String range) {
+        clearSpreadSheetValues(TEST_SHEET_ID, range);
+    }
+
+    public void clearSpreadSheetValues(String id, String range) {
+        ClearValuesRequest clear = new ClearValuesRequest();
+        try {
+            Sheets.Spreadsheets.Values.Clear request = sheets.spreadsheets().values().clear(id, range, clear);
+            request.execute();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+    public List<String> getSpreadSheetValues(String id, String range) {
+        List<String> result = new ArrayList<>();
+        try {
+            ValueRange response = sheets.spreadsheets().values()
+                    .get(id, range)
+                    .execute();
+            List<List<Object>> values = response.getValues();
+            if (values == null || values.isEmpty()) {
+                log.info("No data found.");
+            } else {
+                for (List row : values) {
+                    result.add(row.toString());
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        log.info("Spreadsheet values" + result.toString());
+        return result;
+    }
+
+    public List<Sheet> getSheets(String id) {
+        Spreadsheet spreadsheet = null;
+        try {
+            spreadsheet = sheets.spreadsheets().get(id).execute();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return spreadsheet.getSheets();
+    }
+
+    public void clearSheet(String spreadsheetId, int id) {
+        DeleteSheetRequest clear = new DeleteSheetRequest();
+        clear.setSheetId(id);
+
+        BatchUpdateSpreadsheetRequest req = new BatchUpdateSpreadsheetRequest();
+        req.setRequests(new ArrayList<>());
+        req.getRequests().add(new Request().setDeleteSheet(clear));
+
+        try {
+            sheets.spreadsheets().batchUpdate(spreadsheetId, req).execute();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public Spreadsheet getSpreadSheet(String id) {
```

Comment:
this method is used only with `TEST_SHEET_ID`, can it be refactored to use it by default?

Reply:
same as before